app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/model/SourceDefinition.scala

```diff
@@ -44,11 +44,11 @@ object SourceDefinition extends LazyLogging {
   ) {
     @SuppressWarnings(Array("OptionGet"))
     def toSourceDefinition: SourceDefinition = {
-      if (extent.isDefined && crs.isDefined && cellSize.isDefined) {
+      if (extent.isDefined && crs.isDefined && cellSize.isDefined && extentCrs.isDefined) {
```

Comment:
I don't think. I cannot think of one and I also don't know how we would allow that in the current setup.